src/main/java/org/fundacionjala/pivotal/cucumber/selenium/pages/accounts/SettingAccount.java

```diff
@@ -18,7 +18,7 @@
      * This method doing click on "Ok alert Message" button.
```

Comment:
The comment should be "This method is clicking on the "Ok alert Message" button."